.ci/test.sh

```diff
@@ -310,10 +316,10 @@ matplotlib.use\(\"Agg\"\)\
         'h5py>=3.10' \
         'ipywidgets>=8.1.2' \
         'notebook>=7.1.2'
-    for f in *.py **/*.py; do python $f || exit 1; done  # run all examples
+    for f in *.py **/*.py; do python "$f" || exit 1; done  # run all examples
     cd "$BUILD_DIRECTORY/examples/python-guide/notebooks"
     sed -i'.bak' 's/INTERACTIVE = False/assert False, \\"Interactive mode disabled\\"/' interactive_plot_example.ipynb
-    jupyter nbconvert --ExecutePreprocessor.timeout=180 --to notebook --execute --inplace *.ipynb || exit 1  # run all notebooks
+    jupyter nbconvert --ExecutePreprocessor.timeout=180 --to notebook --execute --inplace ./*.ipynb || exit 1  # run all notebooks
```

Comment:
I support this change for clarity, but for my own understanding... did `shellcheck` warn about it?

Reply:
Yeah it was raising  SC2035 (info): Use ./*glob* or -- *glob* so names with dashes won't become options.

Reply:
oh interesting! Did not know it had that, that's a great check.